templates/cohorts/cohort_details.html

```diff
@@ -241,20 +241,20 @@ <h4 class="panel-title">Selected Filters</h4>
                 <h4 class="panel-title">Details</h4>
             </div>
             <div class="panel-body">
-                <div class="single-line">
+                <div class="row col-md-12 row-spacer-v">
```

Comment:
oh right. I understand now. Okay okay. 

Also, I realize these are small nitpicky things so feel free to ignore my suggestions. I think it's more important to get the patient counts working than dealing with these layout issues. Is that coming in a separate pull?

Thanks!!